pkg/proxy/apis/config/v1alpha1/zz_generated.conversion.go

```diff
@@ -100,7 +99,8 @@ func autoConvert_v1alpha1_KubeProxyConfiguration_To_config_KubeProxyConfiguratio
 	out.EnableProfiling = in.EnableProfiling
 	out.ClusterCIDR = in.ClusterCIDR
 	out.HostnameOverride = in.HostnameOverride
-	if err := configv1alpha1.Convert_v1alpha1_ClientConnectionConfiguration_To_config_ClientConnectionConfiguration(&in.ClientConnection, &out.ClientConnection, s); err != nil {
```

Comment:
@khenidak this is what breaks the unit tests

Reply:
Yes I think this is https://github.com/kubernetes/kubernetes/issues/95803 the reason.